src/components/Report.tsx

```diff
@@ -0,0 +1,54 @@
+import React, { useState } from "react";
+import DatePicker from "react-datepicker";
+import "react-datepicker/dist/react-datepicker.css";
+import "./Report.css";
+import { BsCalendarEvent } from "react-icons/bs";
+
+export default function Report() {
+  const [startDate, setStartDate] = useState<Date | null>(null);
+  const [endDate, setEndDate] = useState<Date | null>(null);
+  return (
+    <div className="report_box">
+      <span className="intro-text"> Enter Dates </span>
+      <span id="icon">
+        <BsCalendarEvent />{" "}
+      </span>
+      <div id="segment">
+        <div id="divTable" className="InsideContent">
+          <table id="logtable">
+            <DatePicker
+              className="date_box"
+              selected={startDate}
+              onChange={(date) => setStartDate(date)}
+              placeholderText="mm/dd/yy"
```

Comment:
The figma design shows the placeholder as this but let's change it to "Start Date" to make it consistent with the other picker.